packages/frontend/components/primitives/range/component.tsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react/display-name */
+import React, { useState } from 'react';
+import { StyledRange } from './styled-range/component';
+
+export const Range = React.forwardRef((ref) => {
+  const [value, setValue] = useState([0, 50000]);
```

Comment:
it make sense to have the component stateless.

UPD: you don't need this component